syllabus/task_space/task_space.py

```diff
@@ -11,6 +11,13 @@ def __init__(self, gym_space: Union[Space, int], tasks=None):
             # Syntactic sugar for discrete space
             gym_space = Discrete(gym_space)
 
+        if isinstance(gym_space, MultiDiscrete):
+            # Syntactic sugar for discrete space
+            gym_space = MultiDiscrete(gym_space)
```

Comment:
```suggestion
        if isinstance(gym_space, tuple):
            # Syntactic sugar for discrete space
            gym_space = ...
```

The point of the syntactic sugar is just to make it easier to initialize a TaskSpace from simple types, so the equivalent syntax for MultiDiscrete would be the user passing a tuple which we use to initialize the MultiDiscrete.